x-pack/plugins/security_solution/public/sourcerer/experimental/readme.md

```diff
@@ -0,0 +1,18 @@
+# Experimental Sourcerer Replacement
+
+## Introduction
+
+This directory is a home for Discovery Components based re-implementation of the Sourcerer.
```

Comment:
What do you mean by `Discovery Components`?

Is it related to `@elastic/kibana-data-discovery`  team ?

Reply:
Yup. We want to use the same picker they have for Discover table.

Reply:
In that case, I think it might make sense for it to be in a seprate package such as `discover-utils` or any other.

Reply:
Oh no, it is specific to Security. What I meant is that I will use some of the things already exported by Discovery.